app/actions/ActionRefiners.scala

```diff
@@ -1,29 +0,0 @@
-package actions
-
-import com.gu.memsub.Subscription
-import play.api.mvc.{ActionFilter, Result, Request}
-import scala.concurrent.ExecutionContext.Implicits.global
-import services.AuthenticationService._
-import services.TouchpointBackend
-import utils.TestUsers.PreSigninTestCookie
-import scalaz.std.scalaFuture._
-import scala.concurrent.Future
-import scalaz.OptionT
-import org.joda.time.DateTime
-
-
-object ActionRefiners {
-
-  def noSubscriptionAction(onSubscription: Subscription => Result): ActionFilter[Request]  = new ActionFilter[Request] {
-    override def filter[A](request: Request[A]): Future[Option[Result]] = {
-      authenticatedUserFor(request).fold[Future[Option[Result]]] {Future.successful(None)} { user =>
-        val tp = TouchpointBackend.forRequest(PreSigninTestCookie, request.cookies)(request).backend
```

Comment:
So this is what we used to use to detect duplicate digipack subs if you were signed in

But for paper we use the same checkout so this is no longer good, we don't want to kick you out just because you're subscribed to digipack